doc/building.rst

```diff
@@ -69,17 +77,40 @@ CMake project definition looks as follows:
     #   find_package(open62541 REQUIRED)
     #   target_link_libraries(main open62541::open62541)
 
-Building with CMake on Windows
+Building with Visual Studio on Windows
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
The underline must/should be as long as the heading, according to rst. Sphinx will still build it but print the warning `Title underline too short`.